site/includes/footerresources.hbs

```diff
@@ -1,17 +1,17 @@
 <!--[if gte IE 9 | !IE ]><!-->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/{{environment.jqueryVersion}}/jquery{{environment.suffix}}.js"></script>
-<script src="{{assets}}/js/wet-boew{{environment.suffix}}.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/{{environment.jqueryVersion}}/jquery.js"></script>
```

Comment:
We would need to move the UseMinAssets to a separate task, it gets really annoying to update it on every theme.